### PR TITLE
PRMP-1035 Remove duplicate stitch lambda call

### PR DESCRIPTION
**Pull request by steph-torres-nhs**

### CODE REVIEW

File: app/src/components/blocks/_lloydGeorge/lloydGeorgeViewRecordStage/LloydGeorgeViewRecordStage.tsx [hunk 1]
```diff
@@ -209,17 +212,19 @@ function LloydGeorgeViewRecordStage({
             <PatientSimpleSummary />
             {!fullScreen ? (
                 <>
-                    {showMenu ? (
+                    <>
```
Comment (NogaNHS) [inline app/src/components/blocks/_lloydGeorge/lloydGeorgeViewRecordStage/LloydGeorgeViewRecordStage.tsx:215]: remove

File: app/src/components/generic/pdfViewer/PdfViewer.tsx [hunk 2]
```diff
@@ -5,7 +5,7 @@ type Props = { fileUrl: String };
 const PdfViewer = ({ fileUrl }: Props) => {
     useEffect(() => {
         const pdfObject = require('pdfobject');
-        pdfObject.embed(fileUrl + '#toolbar=0', '#pdf-viewer');
+        pdfObject.embed(fileUrl + '#toolbar', '#pdf-viewer');
```
Comment (NogaNHS) [inline app/src/components/generic/pdfViewer/PdfViewer.tsx:8]: undo this change

File: app/src/components/generic/recordCard/RecordCard.tsx [hunk 3]
```diff
@@ -26,27 +23,21 @@ function RecordCard({
...
     }, [refreshRecord]);
 
     const Record = () => {
-        if (isLoading) {
-            return (
-                <div className="pl-7">
-                    <ProgressBar status="Loading..." />
-                </div>
-            );
+        if (!cloudFrontUrl) {
```
Comment (AndyFlintNHS) [inline app/src/components/generic/recordCard/RecordCard.tsx:39]: Return null always jumps out to me, could this be replaced with:
`if (cloudFrontUrl) {
  return <PdfViewer fileUrl={cloudFrontUrl} />;
}`

File: app/src/components/generic/recordMenuCard/RecordMenuCard.test.tsx [hunk 4]
```diff
@@ -49,6 +49,8 @@ const mockLinks: Array<LGRecordActionLink> = [
     },
 ];
 
+const className = 'lloydgeorge_record-stage_flex-row';
```
Comment (AndyFlintNHS) [inline app/src/components/generic/recordMenuCard/RecordMenuCard.test.tsx:52]: Nitpicky one, would it be worth referring to this as something that indicates that it's a formatting thing? Something about a variable called `className` jumps out to me.

**Review by AndyFlintNHS — CHANGES_REQUESTED** [review]
Comments listed individually

File: app/src/pages/lloydGeorgeRecordPage/LloydGeorgeRecordPage.tsx [hunk 5]
```diff
@@ -122,6 +124,7 @@ function LloydGeorgeRecordPage() {
                             numberOfFiles={numberOfFiles}
                             refreshRecord={refreshRecord}
                             cloudFrontUrl={cloudFrontUrl}
+                            showMenu={showMenu}
                         />
```
Comment (RioKnightleyNHS) [inline app/src/pages/lloydGeorgeRecordPage/LloydGeorgeRecordPage.tsx:128]: Im not seeing any new unit tests for the LloydGeorgeRecordPage to reflect this new prop. Need tests to check what happens when this is toggled

File: app/src/components/generic/recordMenuCard/RecordMenuCard.tsx [hunk 6]
```diff
@@ -15,33 +17,38 @@ type SubSectionProps = {
...
     const updateActions = recordLinks.filter((link) => link.type === RECORD_ACTION.UPDATE);
     const downloadActions = recordLinks.filter((link) => link.type === RECORD_ACTION.DOWNLOAD);
 
     if (recordLinks.length === 0) {
         return <></>;
     }
 
+    if (!showMenu) {
+        return null;
+    }
```
Comment (RioKnightleyNHS) [inline app/src/components/generic/recordMenuCard/RecordMenuCard.tsx:30]: Id say here to decide between returning null or a fragment so its consistent

File: app/src/components/generic/recordMenuCard/RecordMenuCard.test.tsx (hunk 4 above)
Comment (RioKnightleyNHS) [reply app/src/components/generic/recordMenuCard/RecordMenuCard.test.tsx:52]: This classname can probably become a default prop of className in the actualy component, unless its overriden, so we dont need to paste a default one for every test

**Review by RioKnightleyNHS — CHANGES_REQUESTED** [review]

File: app/src/pages/lloydGeorgeRecordPage/LloydGeorgeRecordPage.tsx (hunk 5 above)
Comment (NogaNHS) [reply app/src/pages/lloydGeorgeRecordPage/LloydGeorgeRecordPage.tsx:128]: there is a test for `Does not render the MenuCard if showMenu is false` in RecordMenuCard, do you think we need to tests the LloydGeorgeRecordPage ?

File: app/src/components/generic/recordMenuCard/RecordMenuCard.tsx [hunk 7]
```diff
@@ -15,33 +17,38 @@ type SubSectionProps = {
     setStage: Dispatch<SetStateAction<LG_RECORD_STAGE>>;
 };
 
-function RecordMenuCard({ recordLinks, setStage }: Props) {
+function RecordMenuCard({ recordLinks, setStage, showMenu, className }: Props) {
     const updateActions = recordLinks.filter((link) => link.type === RECORD_ACTION.UPDATE);
```
Comment (RioKnightleyNHS) [inline app/src/components/generic/recordMenuCard/RecordMenuCard.tsx:21]: Add the default classname here in the props deconstruction props = 'classnamestring'. This means it still has a prop it can pass but if nothing is passed itll use this default string. It prevents needing to type it in the test cases.

**Review by RioKnightleyNHS — CHANGES_REQUESTED** [review]

File: app/src/components/generic/recordCard/RecordCard.tsx (hunk 3 above)
Comment (AndyFlintNHS) [reply app/src/components/generic/recordCard/RecordCard.tsx]: Discussed this together and came up with a ternary.

File: app/src/pages/lloydGeorgeRecordPage/LloydGeorgeRecordPage.tsx (hunk 5 above)
Comment (RioKnightleyNHS) [reply app/src/pages/lloydGeorgeRecordPage/LloydGeorgeRecordPage.tsx:128]: generally speaking we should but as there is already so much bloat in these tests we are probably fine

**Review by AndyFlintNHS — DISMISSED** [review]

**Review by RioKnightleyNHS — DISMISSED** [review]

**Review by abid-nhs — APPROVED** [review]

**Review by RioKnightleyNHS — APPROVED** [review]

**Review by NogaNHS — APPROVED** [review]

**Review by abid-nhs — APPROVED** [review]
